### PR TITLE
Add multiline messages and messages from file

### PR DESCRIPTION
**Pull request by avanier**
This PR adds the following:

- The ability to take a message from a file or from the `param`.
- The ability to send multiline messages as a sequence of messages
- Adds some safety features to the integration test script
- Version pins Alpine in the Dockefile
- Cleans the image on build to expand compatibility to OCI image builders that don't respect `.dockerignore`

### CODE REVIEW

**Comment by flavorjones** [comment]
Thanks for submitting this! I'm going to make some comments on the commits.

**Review by flavorjones — CHANGES_REQUESTED** [review]
Can I ask that commit 1ea78e47489eb00f5cd0fd5aca327cdd8724d770 go into a separate PR? I'd like to understand more deeply the problem you're solving, and this seems unrelated to the feature/functional parts of the PR.

File: pkg/irc/irc.go [hunk 1]
```diff
@@ -8,11 +8,12 @@ import (
 	"io/ioutil"
 	"log"
 	"os"
+	"strings"
 
 	"github.com/Elemental-IRCd/irc"
 )
 
-const VERSION = "v1.2.0"
+const VERSION = "v1.3.0"
```
Comment (flavorjones) [inline pkg/irc/irc.go]: I'd prefer if the version wasn't bumped in a PR - can I ask that you please remove this line?

**Comment by avanier** [comment]
> Can I ask that commit [1ea78e4](https://github.com/flavorjones/irc-notification-resource/commit/1ea78e47489eb00f5cd0fd5aca327cdd8724d770) go into a separate PR? I'd like to understand more deeply the problem you're solving, and this seems unrelated to the feature/functional parts of the PR.

The situation I was facing is I discovered that `podman` and `buildah` would ignore the the `.dockerignore` and stick my locally built artifacts in the build context and `make test` would fail with `file not found` due to dynamic linking errors.

As for the pinning of the image versions, it _"fixes nothing"_ I was on a drive-by fix binge and it went in.

**Comment by avanier** [comment]
The Docker image commit has been moved to #17 .

**Comment by avanier** [comment]
Bump

**Comment by flavorjones** [comment]
@avanier Thanks for your patience. It's been hard to make time for open-source work lately. I'll schedule some time to work on this in the next few days.

**Comment by flavorjones** [comment]
I've added some test coverage and some other small changes, and merged this manually. Thank you for your contribution! I'll update the CHANGELOG to credit you and ship an update now.

**Comment by flavorjones** [comment]
v1.3.0 has been shipped with these features. Thanks again!

**Pull request merged**